charts/camunda-platform-alpha/templates/identity/configmap.yaml

```diff
@@ -155,7 +165,7 @@ data:
         webmodeler:
           applications:
             - name: "Web Modeler"
-              id: ${CAMUNDA_WEBMODELER_CLIENT_ID:${KEYCLOAK_INIT_WEBMODELER_CLIENT_ID:web-modeler}}
+              id: ${CAMUNDA_WEBMODELER_CLIENT_ID:${VALUES_KEYCLOAK_INIT_WEBMODELER_CLIENT_ID:web-modeler}}
```

Comment:
why did these env vars get renamed ?

Reply:
To make sure it's clear, those are Helm vars, not the app vars (someone will try to use them to override the values in the app but they are just used as env vars no spring config env vars).

Reply:
That makes sense